include/robot_localization/ros_filter_bias_estimator.h

```diff
@@ -35,9 +35,10 @@ class RosFilterBiasEstimator {
                                const double max_variance,
                                const double alpha,
                                const double max_divergence = -1.0,
-                               const int max_num_divergences = 0,
+                               const int16_t max_num_divergences = 0,
```

Comment:
Not a critique, but a moment for me to learn. Is the idea behind moving these variables from `int` to`int16_t`, `int32_t` etc. to maintain precision?

Reply:
This was actually me catching myself from the previous version. I strive to always define the size of ints because different machines recognize "int" as different sizes (e.g. some will see it as int16_t versus some as int32_t). Since I had not set this up correctly previously, I fixed it this round.